### PR TITLE
Exit PiP throws InvalidStateError if no pip video

### PR DESCRIPTION
**Pull request by beaufortfrancois**
WDYT?

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -175,15 +175,17 @@ state that is considered not compatible with Picture-in-Picture.
 When the <dfn>exit Picture-in-Picture algorithm</dfn> is invoked,
 the user agent MUST run the following steps:
 
-1. Let |pipWindow| be the current Picture-in-Picture window.
-2. Let |pipVideo| be the video contained in the |pipWindow|.
-3. Let |video| be the source video of |pipVideo| frames.
-4. Close |pipWindow|.
-5. If |pipWindow| can't be closed, throw a {{UnknownError}} and abort these
+1. If {{pictureInPictureElement}} is null, throw a {{NotFoundError}} and abort
```
Comment (mounirlamouri) [inline index.bs]: WDYT of `InvalidStateError`?

File: index.bs [hunk 2]
```diff
@@ -175,15 +175,17 @@ state that is considered not compatible with Picture-in-Picture.
...
+    these steps.
+2. Let |pipWindow| be the current Picture-in-Picture window.
+3. Let |pipVideo| be the video contained in the |pipWindow|.
+4. Let |video| be the source video of |pipVideo| frames.
+5. Close |pipWindow|.
+6. If |pipWindow| can't be closed, throw a {{UnknownError}} and abort these
```
Comment (mounirlamouri) [inline index.bs]: Is it something we know can happen?

File: index.bs (hunk 1 above)
Comment (beaufortfrancois) [reply index.bs]: LGTM

File: index.bs (hunk 2 above)
Comment (beaufortfrancois) [reply index.bs]: I guess everything can happen but I don't have anything in mind that can cause a |pipWindow| to not close.

Comment (mounirlamouri) [reply index.bs]: I think we should drop this. Anything "unexpected" would be handled by the implementations. If we don't have real cases justifying this to happen, we shouldn't worry about it.

Comment (beaufortfrancois) [reply index.bs]: Noted.

**Comment by beaufortfrancois** [comment]
@mounirlamouri Done.

File: index.bs [hunk 3]
```diff
@@ -175,12 +175,12 @@ state that is considered not compatible with Picture-in-Picture.
...
-2. Let |pipVideo| be the video contained in the |pipWindow|.
-3. Let |video| be the source video of |pipVideo| frames.
-4. Close |pipWindow|.
-5. If |pipWindow| can't be closed, throw a {{UnknownError}} and abort these
-    steps.
+1. If {{pictureInPictureElement}} is null, throw a {{InvalidStateError}} and abort
+    these steps.
+2. Let |pipWindow| be the current Picture-in-Picture window.
+3. Let |pipVideo| be the video contained in the |pipWindow|.
+4. Let |video| be the source video of |pipVideo| frames.
```
Comment (mounirlamouri) [inline index.bs]: This difference between |pipVideo|, |video| and |pictureInPictureElement| is weak. Probably orthogonal to this change though.

**Pull request merged**